src/plugins/xfconf/README.md

```diff
@@ -0,0 +1,51 @@
+- infos = Information about the xfconf plugin is in keys below
+- infos/author = Richard Stöckl <e11908080@student.tuwien.ac.at>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides = storage/xfconf
+- infos/recommends =
+- infos/placements = postgetstorage presetstorage
+- infos/status = maintained libc configurable experimental unfinished concept limited memleak
```

Comment:
Why unfinished, what is missing? What do you mean with limited? The limitations below seem to be general limitations of xfconf. If you do everything that xfconf can do, you don't need to declare your plugin limited. Please clarify in the README.

Reply:
This is still open.